core/src/main/java/com/nuodb/migrator/globalStore/GlobalStore.java

```diff
@@ -66,32 +79,51 @@ public String alterScript(LoadTable table) {
                     str.append(col.getJdbcType().getTypeName());
                     str.append(' ');
                     String key = col.getSequence().getName().toString();
-                    for (Map.Entry<String, Boolean> entry : generatedValues.entrySet()) {
-                        String entryInHashMap = entry.getKey();
-                        entryInHashMap.replaceAll("^\"|\"$", "");
-                        entryInHashMap = entryInHashMap.replaceAll("^\"|\"$", "");
-                        boolean valueInHashMap = entry.getValue();
-                        if (generatedValues != null) {
+                    if(generatedValues == null) return null;
+//                    else{
+                        for (Map.Entry<String, Boolean> entry : generatedValues.entrySet()) {
+                            String entryInHashMap = entry.getKey();
+                            entryInHashMap.replaceAll("^\"|\"$", "");
```

Comment:
Java `String` is immutable and `String.replaceAll()` returns a new value. So this line is not actually doing anything. You have to assign the result back to `entryInHashMap`.